workspaces/local-cli/src/index.ts

```diff
@@ -1 +1,3 @@
-export {run} from '@oclif/command'
+import * as dotEnv from 'dotenv';
+dotEnv.config();
```

Comment:
I'm not 100% sure if/when this file ever gets run. As far as I can tell, the entry point(s) are in /bin where we call setLogger. So, I imagine local-cli/src/config.ts is always getting the default value. Other than that I'm fine with there being a hardcoded default and env override

Reply:
It seemingly does, at least through `apidev agent:start`. Let me verify!